docs/source/guide/webhook_reference.md

```diff
@@ -0,0 +1,198 @@
+| is_labeled | boolean | Whether or not the task has been labeled. |
+
+### Example payload
+
+```json
+{
+    "action": "Task created",
+    "tasks": [
+        {"id": 1, ...},
+        ...
+    ]
+}
+```
+
+## Task Deleted
+
+Sent when a task is deleted from Label Studio. See how to [set up a webhook for this event](webhooks.html)
+
+### Webhook payload details
+
+| Key | Type | Description |
+| --- | --- | --- | 
+| name | string | Name of the action. In this case, `Task deleted`. | 
+| id | integer | ID of the deleted task. | 
+
+### Example payload 
+
+```json
+{
+    "action": "Task deleted",
+    "tasks": [
+        {"id": 1, ...},
+        ...
+    ]
+}
+```
+
+## Annotation Created
+Sent when an annotation is created for a task in Label Studio. See how to [set up a webhook for this event](webhooks.html)
+
+### Webhook payload details
+
+| Key | Type | Description |
+| --- | --- | --- | 
+| name | string | Name of the action. In this case, `Annotation created`. | 
+| id | integer | ID of the created annotation. | 
+| result | JSON | JSON format of the annotation created. |
+| task | JSON | The task that the annotation was created for. |
+| completed_by | JSON | Email address of the user that created the annotation. |
+| was_cancelled | boolean | Whether or not the annotation is the result of a skipped task, and an empty annotation. |
+| ground_truth | boolean | Enterprise only. Whether or not the annotation is a ground truth. 
+| created_at | datetime | Date and time that the annotation was created. |
+| updated_at | datetime | Date and time that the annotation was last updated. |
+| prediction | JSON dictionary | Details of the prediction viewed at the time of annotation, if one exists. |
+| project | JSON dictionary | All fields related to the associated project. |
+
+### Example payload
+
+```json
+
+
+```
+
+## Annotation Updated
+
+Sent when an annotation is updated. See how to [set up a webhook for this event](webhooks.html)
+
+### Webhook payload details
+
+| Key | Type | Description |
+| --- | --- | --- | 
+| name | string | Name of the action. In this case, `Annotation created`. | 
+| id | integer | ID of the created annotation. | 
+| result | JSON | JSON format of the annotation created. |
+| task | JSON | The task that the annotation was created for. |
+| completed_by | JSON | Email address of the user that created the annotation. |
+| was_cancelled | boolean | Whether or not the annotation is the result of a skipped task, and an empty annotation. |
+| ground_truth | boolean | Enterprise only. Whether or not the annotation is a ground truth. 
+| created_at | datetime | Date and time that the annotation was created. |
+| updated_at | datetime | Date and time that the annotation was last updated. |
+| prediction | JSON dictionary | Details of the prediction viewed at the time of annotation, if one exists. |
+| project | JSON dictionary | All fields related to the associated project. |
```

Comment:
is this correct?

Reply:
@chiganov waiting for all payloads pasted here